libraries/AP_HAL_ChibiOS/hwdef/MatekF405-Wing/hwdef.dat

```diff
@@ -197,6 +199,15 @@ define HAL_GYROFFT_ENABLED 0
 include ../include/save_some_flash.inc
 define AP_GRIPPER_ENABLED 0
 define AP_BATTMON_SYNTHETIC_CURRENT_ENABLED 0
+define AP_BATTMON_SMBUS_ENABLE 0
```

Comment:
Could we look at including minimize_features.inc instead of putting all of these in?  The `MatekF405-CAN` does this. 

Also... did the board overflow on space - do you really need to do this?

Reply:
MatekF405-TE also does this to save space.

MatekF405-Wing without these additional definitions, 
Target         Text (B)  Data (B)  BSS (B)  Total Flash Used (B)  Free Flash (B)
--------------------------------------------------------------------------------
bin/arduplane    963872      2336   128824                966208           16824

with additional definitions, 
--------------------------------------------------------------------------------
bin/arduplane    953496      2268   128888                955764           27268

Reply:
MatekF405-TE also does this to save space.

MatekF405-Wing without these additional definitions, 
Target         Text (B)  Data (B)  BSS (B)  Total Flash Used (B)  Free Flash (B)
--------------------------------------------------------------------------------
bin/arduplane    963872      2336   128824                966208           16824

with additional definitions, 
--------------------------------------------------------------------------------
bin/arduplane    953496      2268   128888                955764           27268

Reply:
MatekF405-TE also does this to save space.

MatekF405-Wing without these additional definitions, 
Target         Text (B)  Data (B)  BSS (B)  Total Flash Used (B)  Free Flash (B)
bin/arduplane    963872      2336   128824                966208           16824

with additional definitions, 
bin/arduplane    953496      2268   128888                955764           27268